File: Data Structures & Algorithms/maximum-subarray/submission-3.cpp
class Solution {
public:
    int maxSubArray(vector<int>& nums) {
        int maxVal=nums[0];

        for(int i=0;i<nums.size();i++) {
            int curSum=nums[i];
            if(curSum>maxVal)maxVal=curSum;
            for(int j=i+1;j<nums.size();j++) {
                curSum+=nums[j];
                if(curSum>maxVal){
                    maxVal=curSum;
                   
                
                 }
                if(curSum<=0)break;
            }
            
           
        }

        return maxVal;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-3.py
from operator import itemgetter
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq={}
        result=[]
        for i in range (len(nums)):
            if nums[i] in freq:
                freq[nums[i]]+=1
            else:
                freq[nums[i]]=1
        
        sorted_freq=dict(sorted(freq.items(),key=itemgetter(1),reverse=True))
        for key,val in sorted_freq.items():
            result.append(key)
        
        return result[:k]